src/main/resources/chains/eth-chain.conf

```diff
@@ -197,6 +218,51 @@
     ]
   }
 
+  # Starting nonce of an empty account. Some networks (like Morden) use different values.
+  account-start-nonce = "0"
+
+  # The ID of the accepted chain
+  chain-id = "1"
+
+  # Custom genesis JSON file path
+  # null value indicates using default genesis definition that matches the main network
+  custom-genesis-file = null
```

Comment:
minor: should we maybe get rid of optional genesis path? (separate task for Piotr or Adiran maybe?)

Reply:
Totally! Here's the task for that: https://jira.iohk.io/browse/ETCM-128